libs/sdk-ui-dashboard/src/presentation/shareDialog/types.ts

```diff
@@ -1,21 +1,32 @@
 // (C) 2019-2021 GoodData Corporation
 import { ComponentType } from "react";
-import { IAccessControlAware } from "@gooddata/sdk-backend-spi";
-import { IAuditableUsers, ObjRef } from "@gooddata/sdk-model";
-import { IShareProps } from "../../types";
+import { IAnalyticalBackend } from "@gooddata/sdk-backend-spi";
+import { ObjRef } from "@gooddata/sdk-model";
+import { ISharedObject, ISharingApplyPayload as Payload } from "@gooddata/sdk-ui-kit";
+import { GoodDataSdkError } from "@gooddata/sdk-ui";
 
 /**
  * @alpha
  */
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
-export interface ISharingApplyPayload extends IShareProps {
+export interface ISharingApplyPayload extends Payload {
```

Comment:
Payload is quite generic. why do we have ISharingApplyPayload and the same in kit?

Reply:
perhaps change the name of this payload object if the name clashes.. seems this is something like extended sharing apply payload or something?

Reply:
Done